go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/jbuchbinder/gopnm v0.0.0-20220507095634-e31f54490ce0 // indirect
 	github.com/kortschak/goroutine v1.1.3 // indirect
 	github.com/rivo/uniseg v0.4.7 // indirect
-	golang.org/x/crypto/x509roots/fallback v0.0.0-20250406160420-959f8f3db0fb // indirect
+	golang.org/x/crypto/x509roots/fallback v0.0.0-20251112184832-bcf6a849efcf // indirect
```

Comment:
given the code doesn’t do https calls updating this is a bit unnecessary (but fine too, I just avoid go get -u ./… most times (and let dependabot deal with updates))